packages/components/src/components/chart/ChartMenu.tsx

```diff
@@ -13,135 +13,183 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React from 'reactn';
+import React from 'react';
 import { DropdownButton, MenuItem, Modal } from 'react-bootstrap';
 import { List } from 'immutable';
 
+import { generateId } from '../../util/utils';
+import { QueryGridModel } from '../base/models/model';
 import { DataViewInfo } from '../../models';
 import { Chart } from './Chart';
 import { setReportId } from '../../actions';
-import { QueryGridModel } from '../base/models/model';
-import { generateId, naturalSort } from '../../util/utils';
+
+interface ChartModalProps {
+    selectedChart: DataViewInfo,
+    model: QueryGridModel,
+    onHide: Function,
+}
+
+class ChartModal extends React.PureComponent<ChartModalProps> {
```

Comment:
This seems separable enough that it should be moved into its own file so that this file remains just the ChartMenu pieces.

Reply:
Done.